From 4dad8af721e72b08c40df2c37558775a24987420 Mon Sep 17 00:00:00 2001
From: Luca Kellermann <mailto.luca.kellermann@gmail.com>
Date: Wed, 15 Apr 2026 23:00:50 +0200
Subject: scandir: remove unused header

stddef.h is unused, commit da88b16a221c9d327e1bfa61dd6f4f08dacce57a
removed the use of offsetof().
---
 src/dirent/scandir.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/dirent/scandir.c b/src/dirent/scandir.c
index 170f9283..4eab99ef 100644
--- a/src/dirent/scandir.c
+++ b/src/dirent/scandir.c
@@ -5,7 +5,6 @@
 #include <stdlib.h>
 #include <stdint.h>
 #include <errno.h>
-#include <stddef.h>
 
 static int wrapper_cmp(const void *v1, const void *v2, void *cmp)
 {
-- 
cgit v1.2.1

